packages/module/src/EmptyDetailsView/EmptyDetailsView.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateActions,
+  EmptyStateFooter,
+  EmptyStateProps
+} from '@patternfly/react-core';
+
+export interface EmptyDetailsViewProps extends Omit<EmptyStateProps, 'children' | 'titleText'> {
+  /** Empty state title text. */
+  title?: string;
+  /** Empty state description - will be rendered inside EmptyStateBody. */
+  description?: React.ReactNode;
+  /** Source path to an icon image. */
+  iconImage?: string;
+  /** Alternative text for icon image if image can't load. */
+  imageAlt?: string;
+  /** Height of an icon image. */
+  imageSize?: string;
+  /** Flag indicating that creation is allowed. */
+  allowCreate?: boolean;
+  /** Button which initiates the creation. */
+  createButton?: React.ReactNode;
+  /** Extra children to render inside EmptyStateFooter. */
+  footerExtraChildren?: React.ReactNode;
```

Comment:
The example provided in the preview shows an empty state with just a single primary action, but it would be good to have an example that shows a secondary action, too.